server/api/min_resolved_ts.go

```diff
@@ -67,19 +70,38 @@ func (h *minResolvedTSHandler) GetStoreMinResolvedTS(w http.ResponseWriter, r *h
 	})
 }
 
-// @Tags     min_resolved_ts
-// @Summary  Get cluster-level min resolved ts.
+// @Tags         min_resolved_ts
+// @Summary      Get cluster-level min resolved ts.
+// @Description  Optionally, if a list of store IDs is provided in the request body,
+// it also returns the min_resolved_ts for the specified stores in a separate map.
 // @Produce  json
 // @Success  200  {array}   minResolvedTS
 // @Failure  500  {string}  string  "PD server failed to proceed the request."
-// @Router   /min-resolved-ts [get]
+// @Router       /min-resolved-ts [get]
 func (h *minResolvedTSHandler) GetMinResolvedTS(w http.ResponseWriter, r *http.Request) {
 	c := h.svr.GetRaftCluster()
-	value := c.GetMinResolvedTS()
+	scopeMinResolvedTS := c.GetMinResolvedTS()
 	persistInterval := c.GetPDServerConfig().MinResolvedTSPersistenceInterval
+
+	var storesMinResolvedTS map[uint64]uint64
+	if b, err := io.ReadAll(r.Body); err == nil && len(b) != 0 {
```

Comment:
Why not use query parameters?

Reply:
I think it's too weird when 100 stores need to be queried by parameters...

Reply:
it's also weird `GET` request with body paameters.

Reply:
done